docker-compose-dev.yml

```diff
@@ -29,6 +29,8 @@ services:
     build:
       context: .
       dockerfile: ./docker/dev/django/Dockerfile
+      args:
+          REALM: "${REALM:-development}"
```

Comment:
Nice underrated bash syntax. Means "get REALM and default to development if none".

S'utilise aussi typiquement pour un passage de branche en paramètre avec un default sur master : `BRANCH=${1:-master}`